src/radical/pilot/configs/resource_epsrc.json

```diff
@@ -26,6 +25,42 @@
         "virtenv"                     : "/work/e290/shared/shared_pilot_ve_20150429/",
         "virtenv_mode"                : "use",
         "rp_version"                  : "debug"
+    },
+    "archer_orte": {
+        "description"                 : "The EPSRC Archer Cray XC30 system (https://www.archer.ac.uk/)",
+        "notes"                       : "Always set the ``project`` attribute in the ComputePilotDescription or the pilot will fail.",
+        "schemas"                     : ["ssh"],
+        "mandatory_args"              : ["project"],
+        "ssh"                         :
+        {
+            "job_manager_endpoint"    : "pbspro+ssh://login.archer.ac.uk/",
+            "filesystem_endpoint"     : "sftp://login.archer.ac.uk/"
+        },
+        "default_queue"               : "standard",
+        "lrms"                        : "PBSPRO",
+        "agent_type"                  : "multicore",
+        "agent_config"                : "cray",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
```

Comment:
Does the shell spawner also fail on Archer?  Not asking because I think we should switch, just wonder if you happen to have checked...